ProjHttpsClient.ino

```diff
@@ -0,0 +1,161 @@
+/*********************************************************
+Project:              Solid enabled IoT sensor platform
+Team:                 INSA 5ISS Solid-IoT team
+Author:               Xiaotong XIE
+Function:             POST sensor data to Solid POD https://solid-iot.solidcommunity.net/iot/
+Fingerprints(SHA1):   99:F4:66:89:DE:98:3A:B4:7F:AC:79:62:84:D1:48:CA:FC:CB:22:B5
+State:                Pass-test
+*********************************************************/
+
+#include <ESP8266WiFi.h>
+#include <WiFiClientSecure.h>
+#include <time.h>
+
+#define BAUD_SPEED 115200
+#define PIN_SENSOR A0
+
+const char* ssid = "HUAWEI"; // WiFi name
+const char* password ="xiexiaotong"; // WiFi password
+
+String host = "solid-iot.solidcommunity.net"; // host
+String urlParams = "iot/"; // path
+const char fingerPrint[] PROGMEM = "99:F4:66:89:DE:98:3A:B4:7F:AC:79:62:84:D1:48:CA:FC:CB:22:B5"; // SHA1 SSL finger print of the host
+const int httpsPort = 443; // conventional port
+
+String sensor = "http://insa-toulouse.fr/solid-iot/pollution/sensor-xie"; // URIs W3C:sosa
+String hasSimpleResult = "http://www.w3.org/ns/sosa/hasSimpleResult";
+String madeBySensor = "http://www.w3.org/ns/sosa/madeBySensor";
+String resultTime = "http://www.w3.org/ns/sosa/resultTime";
+String dateTime = "http://www.w3.org/2001/XMLSchema#dateTime";
+
+int timezone = 1 * 3600; // Unit second for NTP
+int i=0;
+
+
+void setup() {
+  delay(1000);
+  
+  // WiFi connection
+  Serial.begin(BAUD_SPEED);
+  while (!Serial);
+  Serial.print("Connecting to ");
+  Serial.print(ssid);
+  WiFi.mode(WIFI_STA);
+  WiFi.begin(ssid,password);
+  while(WiFi.status() != WL_CONNECTED){
+    Serial.print(" .");
+    delay(1000);
+  }
+  Serial.println();
+  Serial.print("Connection is established, ");
+  Serial.print("IP address: ");
+  Serial.println(WiFi.localIP());
+  Serial.println("==============================================================================");
+
+  // Configuration of pin mode
+  pinMode(PIN_SENSOR,INPUT);
+
+  // NTP server
+  Serial.print("Connecting to NTP");
+  configTime(timezone, 0, "pool.ntp.org","time.nist.gov"); // Fetch time from NTP server
+  while(!time(nullptr)){
+    Serial.print(" .");
+    delay(1000);   
+  }
+  Serial.println();
+  Serial.println("Connection with NTP is established!");
+  Serial.print("Syncing with NTP");
+  while(i<20){
+    delay(1000);
+    Serial.print(" .");
+    i++;
+  }
```

Comment:
Does synchronizing with NTP just require a bit of time ? If so, a comment could be good here.